dht_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
Needs to be gxed (I don't think it is right now).

Reply:
Doing that now.

Reply:
Ah, it already was: https://github.com/gxed/testify. Gonna update that to the latest gomod release, make a gx release, and update `package.json`. Thanks for spotting this, @Kubuxu .

Reply:
Thank you both.

Reply:
@raulk What's required to get that in? Is it just to pass the old gx-based CI? Do we care if the go mod CI is about to get merged?

Reply:
Yes, we care *a lot*. Everything must build with gx. IPFS and other downstreams use it.

Here's the process I followed to update a gxed repo from the original upstream, and make it available to this build:

1. I merged all commits from upstream (stretchr/testify) to gxed/testify up until their latest tag (v1.3.0).
2. Then I made a gx release: `gx release minor`. I actually did this several times until the resulting version was 1.3.0, to line up version numbers (theirs with gx's). I discarded the intermediate commits.
3. I pinned the resulting hash via our pinbot (ping me for more details). This ensures the dependency will always be available for build jobs.
4. `gx import <hash>`, where hash in this case was QmYNszQ8MWA7o9qhjTZrp6R9DXNn1fvTZufyb6LmKHcgcr.

Here's the commit; it should build with gx (and at least we should have one pipeline in our CI ensuring that's the case -- do we?): b96faae

Reply:
Jenkins builds with gx, phew!

Reply:
Done.